util/tlgen/generate.py

```diff
@@ -23,14 +23,14 @@ def generate(xbar):  #xbar: Xbar -> str
         filename=resource_filename('tlgen', 'xbar.pkg.sv.tpl'))
     #xbar_dv_tpl = Template(
```

Comment:
nit: is this still needed?

Reply:
It's to generate a simple TB used by @eunchan. Perhaps still keep it there as well as `xbar.dv.sv.tpl` in case Eunchan wants to use it again.